.circleci/config.yml

```diff
@@ -97,10 +97,30 @@ jobs:
             make integration-test
       - go/save-cache
 
+  release:
+    docker:
+      - image: circleci/golang:1.12.10
+    working_directory: /go/src/github.com/dragonflyoss/Dragonfly
+    steps:
+      - checkout
+      - go/load-cache
+      - run: curl -sL https://git.io/goreleaser | bash
+      - go/save-cache
+
 workflows:
   version: 2
   ci:
     jobs:
       - linters-check
       - unit-test-golang
       - api-integration-test
+
+  release:
+    jobs:
+      - release:
+          # Only run this job on git tag pushes
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^[0-9]+(\.[0-9]+){2}(-.+|[^-.]*)$/
```

Comment:
tag name starts with char 'v', but version name does not start with 'v'.

Reply:
Done